app/controllers/sessions_controller.rb

```diff
@@ -59,4 +62,12 @@ def check_organization
   def return_to
     request.env['omniauth.origin'] || root_path
   end
+
+  def auto_confirm_development_user auth, user
```

Comment:
Could you add parens here and to the method call above? I try to use parens for method with arguments.